ops/terraform/remote_files/scripts/apply-http-allowlist.sh

```diff
@@ -8,12 +8,12 @@ set -euxo pipefail
 
 ALLOWLIST=./http-domain-allowlist.txt
 
-TYPE=$(echo "$BACALHAU_JOB_SELECTION_PROBE_DATA" | jq -r '.spec.Network.Type')
+TYPE=$(echo "$BACALHAU_JOB_SELECTION_PROBE_DATA" | jq -r '.Job.Tasks[0].Network.Type')
 test "$TYPE" = 'HTTP' || test "$TYPE" = 'None'
```

Comment:
This will break existing allow-listing user scripts that depend on parsing the previous job schema. Do we have enough users that use this feature for this to be a concern?